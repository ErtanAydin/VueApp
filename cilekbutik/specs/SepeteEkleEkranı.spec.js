import { mount, RouterLinkStub } from "@vue/test-utils";
import SepeteEkleEkranı from "@/components/SepeteEkleEkranı";

describe("SepeteEkleEkranı", () => {
  test("mounts properly", () => {
    const wrapper = SepeteEkleEkranı;
    expect(wrapper).toBeTruthy();
  });

  test("renders properly", () => {
    const wrapper = mount(SepeteEkleEkranı, { 
      stubs: {
        NuxtLink: RouterLinkStub
      }
    });
    expect(wrapper.html()).toMatchSnapshot();
  });
});
describe ('data', () => {
  test('data test', () => {
    const wrapper = mount(SepeteEkleEkranı,{
    stubs: {
      NuxtLink: RouterLinkStub
    }});
    expect(wrapper.vm.UrunOzellikleri).not.toBe(undefined)
    expect(wrapper.vm.counter).toBe(1)
  })
})

describe("methods", () => {
  test("AdetArttır", () => {
    const wrapper = mount(SepeteEkleEkranı,{
      stubs: {
        NuxtLink: RouterLinkStub
      }});
      wrapper.vm.AdetArttır()
      expect(wrapper.vm.counter).toBe(2)
  });
  test("AdetAzalt", () => {
    const wrapper = mount(SepeteEkleEkranı,{
      stubs: {
        NuxtLink: RouterLinkStub
      }});
      wrapper.vm.AdetArttır()
      expect(wrapper.vm.counter).toBe(2)
      wrapper.vm.AdetAzalt()
      expect(wrapper.vm.counter).toBe(1)
      wrapper.vm.AdetAzalt()
      expect(wrapper.vm.counter).toBe(1)
  });
});