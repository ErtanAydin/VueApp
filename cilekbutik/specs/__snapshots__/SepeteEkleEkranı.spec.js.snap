// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SepeteEkleEkranı renders properly 1`] = `
<div class="UrunEkranıDiv">
  <div class="UrunEkranı">
    <div class="UrunEkranıImage"><a href="#"><img src="https://www.cilekbutik.net/cdn/2/700/700/images/urunler/5fa6a282db005-1368.jpg" alt="" class="UrunImage"> <img src="https://www.cilekbutik.net/cdn/2/100/100/images/urunler/5fa6a282db005-1368.jpg" alt="" class="UrunKucukImage"></a></div>
    <div class="UrunEkranıBilgi">
      <h3 class="UrunIsim"> CEP KAPAKLI KOT CEKET </h3>
      <p class="UrunBilgi"> Ürün Kodu:</p>
      <p class="UrunBilgi"> Barkod: 6242005630101</p>
      <p class="UrunBilgi"> Kategori: <a class="KategoriLink">Ceket</a></p>
      <p class="UrunBilgi"> Fiyat</p> <b class="UrunFiyat">90,00 TL</b>
      <div class="BedenlerDiv">
        <p class="UrunBeden"> BEDEN:</p>
        <div class="BedenKonum"><button class="Bedenler"><b>S</b></button><button class="Bedenler"><b>M</b></button><button class="Bedenler"><b>L</b></button></div>
      </div>
      <div class="Görünüm"><button class="AdetAzaltButonu">-</button> <input class="AdetInput"> <button class="AdetArttırButonu">+</button> <button class="SepeteEkleButon"> Sepete Ekle</button><img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZlcnNpb249IjEuMSIgeG1sbnM6eGxpbms9Imh0dHA6Ly93d3cudzMub3JnLzE5OTkveGxpbmsiIHhtbG5zOnN2Z2pzPSJodHRwOi8vc3ZnanMuY29tL3N2Z2pzIiB3aWR0aD0iNTEyIiBoZWlnaHQ9IjUxMiIgeD0iMCIgeT0iMCIgdmlld0JveD0iMCAwIDQ1Ni45MzgwOSA0NTYiIHN0eWxlPSJlbmFibGUtYmFja2dyb3VuZDpuZXcgMCAwIDUxMiA1MTIiIHhtbDpzcGFjZT0icHJlc2VydmUiIGNsYXNzPSIiPjxnPjxwYXRoIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgZD0ibTQ0OS45NDE0MDYgODIuNDY4NzVoLTMzNy41NzAzMTJsLTcuMTUyMzQ0LTQ0LjkyOTY4OGMtMy40MjE4NzUtMjEuNDc2NTYyLTIxLjE0ODQzOC0zNy4wNzAzMTItNDIuMTYwMTU2LTM3LjA3MDMxMmgtNTYuMDU4NTk0Yy0zLjg2NzE4OCAwLTcgMy4xMzI4MTItNyA3czMuMTMyODEyIDcgNyA3aDU2LjA1ODU5NGMxNC4wODU5MzcgMCAyNiAxMC44MDg1OTQgMjguMzMyMDMxIDI1LjQ0OTIxOWw0Ni45Njg3NSAyOTUuMTI1YzEuMjM0Mzc1IDkuMzI4MTI1IDkuMTQwNjI1IDE2LjMyODEyNSAxOC41NTA3ODEgMTYuNDI1NzgxaDEyLjkxMDE1NmMzLjM1NTQ2OSAxNyAxOC4yNDYwOTQgMzAuNjE3MTg4IDM2LjA2NjQwNyAzMC42MTcxODggMTcuODE2NDA2IDAgMzIuNzEwOTM3LTEzLjYxNzE4OCAzNi4wNjI1LTMwLjYxNzE4OGg5Ny41MjczNDNjMy4zNTE1NjMgMTcgMTguMjQ2MDk0IDMwLjYyODkwNiAzNi4wNjI1IDMwLjYyODkwNiAyMC4yNzczNDQgMCAzNi43NzM0MzgtMTcuMDcwMzEyIDM2Ljc3MzQzOC0zOC4wNTQ2ODdzLTE2LjUtMzguMDYyNS0zNi43NzM0MzgtMzguMDYyNWMtMTguMTEzMjgxIDAtMzMuMTk5MjE4IDEzLjQ4ODI4MS0zNi4yMTQ4NDMgMzEuNDg4MjgxaC05Ny4yMjI2NTdjLTMuMDE1NjI0LTE4LTE4LjEwMTU2Mi0zMS41LTM2LjIxNDg0My0zMS41LTE4LjExNzE4OCAwLTMzLjIwMzEyNSAxMy41LTM2LjIxNDg0NCAzMS41aC0xMi41NzAzMTNjLTIuNDg4MjgxLS4wMjczNDQtNC41NzQyMTgtMS44ODY3MTktNC44ODY3MTgtNC4zNTE1NjItLjAwNzgxMy0uMDQyOTY5LS4wMTU2MjUuMDU4NTkzLS4wMTk1MzIuMDE1NjI0bC04LjcyMjY1Ni01NC42NjQwNjJoMjM5Ljc0MjE4OGMxNy45MDYyNS4wMDM5MDYgMzMuNDIxODc1LTEyLjQwMjM0NCAzNy4zNTU0NjgtMjkuODcxMDk0bDM2LjE4NzUtMTU3LjIyMjY1NmMuNDgwNDY5LTIuMTI4OTA2LS4wMTE3MTgtNC4zNTkzNzUtMS4zNDM3NS02LjA4OTg0NC0xLjMwODU5My0xLjcxMDkzNy0zLjMxNjQwNi0yLjc0NjA5NC01LjQ3MjY1Ni0yLjgxNjQwNnptLTE3MS4zMjQyMTggMTcyLjgwMDc4MWMtNDAuOTA2MjUgMC03NC4wNjY0MDctMzMuMTYwMTU2LTc0LjA2NjQwNy03NC4wNjY0MDYtLjAwMzkwNi00MC45MDYyNSAzMy4xNjAxNTctNzQuMDY2NDA2IDc0LjA2MjUtNzQuMDY2NDA2IDQwLjkwNjI1IDAgNzQuMDY2NDA3IDMzLjE2MDE1NiA3NC4wNjY0MDcgNzQuMDY2NDA2LS4wNDY4NzYgNDAuODg2NzE5LTMzLjE3OTY4OCA3NC4wMTk1MzEtNzQuMDYyNSA3NC4wNjY0MDZ6bTAgMCIgZmlsbD0iI2ZmZmZmZiIgZGF0YS1vcmlnaW5hbD0iIzAwMDAwMCIgc3R5bGU9IiIgY2xhc3M9IiI+PC9wYXRoPjxwYXRoIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgZD0ibTI3OC42MTcxODggMTIxLjE0MDYyNWMtMzMuMTc1NzgyIDAtNjAuMDY2NDA3IDI2Ljg5NDUzMS02MC4wNjY0MDcgNjAuMDY2NDA2LS4wMDM5MDYgMzMuMTc1NzgxIDI2Ljg5MDYyNSA2MC4wNjY0MDcgNjAuMDY2NDA3IDYwLjA2NjQwNyAzMy4xNzE4NzQgMCA2MC4wNjI1LTI2Ljg5NDUzMiA2MC4wNjI1LTYwLjA2NjQwNy0uMDM1MTU3LTMzLjE1NjI1LTI2LjkwNjI1LTYwLjAyNzM0My02MC4wNjI1LTYwLjA2NjQwNnptNDUuOTE0MDYyIDU4LjMyODEyNWMwIDMuODY3MTg4LTMuMTMyODEyIDctNyA3aC0zMi4wNTg1OTR2MzEuOTcyNjU2YzAgMy44NjcxODgtMy4xMzY3MTggNy03IDctMy44NjcxODcgMC03LTMuMTMyODEyLTctN3YtMzEuOTcyNjU2aC0zMS43NzM0MzdjLTMuODY3MTg4IDAtNy0zLjEzMjgxMi03LTdzMy4xMzI4MTItNyA3LTdoMzEuNzczNDM3di0zMS44NTkzNzVjMC0zLjg2NzE4NyAzLjEzMjgxMy03IDctNyAzLjg2MzI4MiAwIDcgMy4xMzI4MTMgNyA3djMxLjg1OTM3NWgzMi4wNTg1OTRjMy44NjcxODggMCA3IDMuMTMyODEyIDcgN3ptMCAwIiBmaWxsPSIjZmZmZmZmIiBkYXRhLW9yaWdpbmFsPSIjMDAwMDAwIiBzdHlsZT0iIiBjbGFzcz0iIj48L3BhdGg+PC9nPjwvc3ZnPg==" class="IconBoyut"> <a><button class="HemenAlButon"><b class="TLicon">₺</b> Hemen Al</button></a> <button class="WhatsappButon"> WHATSAPP İLE SİPARİŞ VER</button> <img src="https://i.hizliresim.com/p5WbED.png" class="Iconkonum"></div> <a href="#" class="Favoriler"><label class="BuyukKalp">♡</label> Favorilerime ekle</a> <img src="https://www.cilekbutik.net/template/smart/default/assets/images/yerli-uretim.png?v=1578595504" class="YerliUretimBoyut">
      <div class="HızlıGönderi"><img src="https://i.hizliresim.com/6VevqG.png" class="DivIcon"> <b class="DivYazı">Hızlı Gönderi</b></div>
      <div class="GüvenliAlışveriş"><img src="https://i.hizliresim.com/9G1N2Y.png" class="DivIcon2"> <b class="DivYazı2">Güvenli Alışveriş</b></div>
      <div class="Çizgi"></div>
      <div class="DivDisplay"><button class="Twitter"><img src="https://i.hizliresim.com/9V5PHm.png"></button> <button class="Facebook"><img src="https://i.hizliresim.com/rbSj77.png"></button> <button class="Linkedin"><img src="https://i.hizliresim.com/7UdmbB.png"></button> <button class="Whatsapp"><img src="https://i.hizliresim.com/p5WbED.png" class="boyut"></button></div>
    </div>
  </div>
</div>
`;
